src/app/static/package.json

```diff
@@ -3,9 +3,6 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "jest": {
```

Comment:
Was getting warnings about this being duplicated with jest config so have removed here